File: count_1000.cpp
#include <iostream>
#include <chrono>

using namespace std;

int main(){
    auto start = chrono::high_resolution_clock::now();

    int count = 0;

    for (long long int i = 0; i < 1'000'000'000'000; ++i) {
        count += 1;
    }

    auto end = chrono::high_resolution_clock::now();

    // Duration
    auto duration = chrono::duration_cast<chrono::milliseconds>(end - start);

    cout << "We reached " << count << " iterations" << "\n";
    cout << "Time taken: " << duration.count() << "ms\n";
}

File: count_1000.py
import time
from datetime import timedelta

start_time = time.time()

# Count to 100 million
for i in range(100_000_000):
    pass

end_time = time.time()

# Calculate the duration
duration = timedelta(seconds=end_time - start_time)

print(f"Time taken: {duration}")
